### ISSUE TITLE
[BUG] Dependencies incompatible with latest Phoenix.

### ISSUE THREAD

**Issue by mveytsman** [akoutmos/prom_ex#182]
I get the following issue when upgrading Phoenix to `1.7.0-rc.0` on a project with prom_ex:

```
Failed to use "plug_cowboy" (version 2.6.0) because
  phoenix (version 1.7.0-rc.0) requires ~> 2.6
  prom_ex (version 1.7.1) requires ~> 2.5.1
  mix.lock specifies 2.6.0
```

Could we have a release with an updated plug_cowboy dependency?

I see there's a PR https://github.com/akoutmos/prom_ex/pull/162 which loosens version requirements which will solve this.

**Comment by mikl** [comment]
Just a FYI for anyone struggling with this, you can override this resolution problem by adding this to your `mix.exs`:

```elixir
{:plug_cowboy, "~> 2.6", override: true},
```

**Comment by tmartin8080** [comment]
Closing, addressed in #192
